dags/infra/s3_buckets.py

```diff
@@ -17,6 +17,14 @@
     "deafrica_sentinel_2", "deafrica-sentinel-2"
 )
 
+SENTINEL_2_SERVICES_BUCKET = Variable.get(
+    "deafrica_sentinel_2_services", "deafrica-services"
+)
+
+SENTINEL_2_SYNC_BUCKET_NAME = Variable.get(
+    "sentinel_2_bucket_name", "deafrica-sentinel-2-dev-sync"
```

Comment:
Why the default value is dev bucket where as all others are pds?

Reply:
@NikitaGandhi this one I wasn't sure what's the PROD name, and because I won't use it in the first moment I didn't give the attention that it needed. Good catch, I'll change.
Thank you